extension/test/projectBuilder.test.ts

```diff
@@ -4,36 +4,39 @@ import * as deepEql from "deep-eql";
+        `${xcbuildPath} --exportRuntimes --compilationmode=Debug --env=Dev --output="${rootPath}${path.sep}XCR" --project=${xcmlPath}`,
+        `${xcbuildPath} --compilationmode=Debug --exportInterface --env=Dev --output="${rootPath}${path.sep}output" --project=${xcmlPath}`
+    ],
+    linux: [
+        `mono ${xcbuildPath} --compilationmode=Debug --build --env=Dev --vs=VS2015 --project=${xcmlPath} --monoPath=“${monoFacadesPath}” --framework=Framework452 `,
+        `mono ${xcbuildPath} --exportRuntimes --compilationmode=Debug --env=Dev --output="${rootPath}${path.sep}XCR" --project=${xcmlPath}`,
+        `mono ${xcbuildPath} --compilationmode=Debug --exportInterface --env=Dev --output="${rootPath}${path.sep}output" --project=${xcmlPath}`
+    ]
+};
+
 describe("projectBuilder test", () => {
     it(`Given a wrongCxmlPath, getCommands should return undefined`, () => {
         const wrongCxmlPath = "";
         const commands = getCommands(xcbuildPath, wrongCxmlPath, rootPath, undefined, windowsPlaform);
         deepEql(commands, undefined).should.equal(true);
     });
 
-    it(`Given a the right argument in a windows platform, getCommands should return the right commands`, () => {
-        let buildCommand = `${xcbuildPath} --compilationmode=Debug --build --env=Dev --vs=VS2015 --project=${xcmlPath}`;
-        let exportRuntimesCommand = `${xcbuildPath} --exportRuntimes --compilationmode=Debug --env=Dev --output="${rootPath}${path.sep}XCR" --project=${xcmlPath}`;
-        let exportInterfaceCommand = `${xcbuildPath} --compilationmode=Debug --exportInterface --env=Dev --output="${rootPath}${path.sep}output" --project=${xcmlPath}`;
-        const expectedCommands = [buildCommand, exportRuntimesCommand, exportInterfaceCommand];
-        const commands = getCommands(xcbuildPath, rootPath, xcmlPath, undefined, windowsPlaform);
-        deepEql(commands, expectedCommands).should.equal(true);
-    });
-
-    it(`Given a the right argument in a linux platform, getCommands should return the right commands`, () => {
-        const buildCommand = `mono ${xcbuildPath} --compilationmode=Debug --build --env=Dev --vs=VS2015 --project=${xcmlPath} --monoPath=“${monoFacadesPath}” --framework=Framework452 `;
-        const exportRuntimesCommand = `mono ${xcbuildPath} --exportRuntimes --compilationmode=Debug --env=Dev --output="${rootPath}${path.sep}XCR" --project=${xcmlPath}`;
-        const exportInterfaceCommand = `mono ${xcbuildPath} --compilationmode=Debug --exportInterface --env=Dev --output="${rootPath}${path.sep}output" --project=${xcmlPath}`;
-        const expectedCommands = [buildCommand, exportRuntimesCommand, exportInterfaceCommand];
-        const commands = getCommands(xcbuildPath, rootPath, xcmlPath, monoFacadesPath, linuxPlaform);
-        deepEql(commands, expectedCommands).should.equal(true);
+    Object.keys(expectedCommands).forEach(platform => {
```

Comment:
Weird way of defining parametrized tests
See this example [https://github.com/xcomponent/vscode-xcomponent/blob/master/extension/test/extension.stateDataCompletion.test.ts#L25](https://github.com/xcomponent/vscode-xcomponent/blob/master/extension/test/extension.stateDataCompletion.test.ts#L25)